README.md

```diff
@@ -106,52 +147,43 @@ for example in examples:
     result = circuit.encrypt_run_decrypt(*example)
     print(f"Evaluation of {' + '.join(map(str, example))} homomorphically = {result}")
 ```
+*This example is explained in more detail [in this part of the documentation](https://docs.zama.ai/concrete/getting-started/quick_start).*
 
-## Documentation
-
-Full, comprehensive documentation is available at [https://docs.zama.ai/concrete](https://docs.zama.ai/concrete).
-
-## Target users
-
-Concrete is a generic library that supports a variety of use cases. Because of this flexibility,
-it doesn't provide primitives for specific use cases.
-
-If you have a specific use case, or a specific field of computation, you may want to build abstractions on top of Concrete.
+<p align="right">
+  <a href="#table-of-contents" > ↑ Back to top </a> 
+</p>
 
-One such example is [Concrete ML](https://github.com/zama-ai/concrete-ml), which is built on top of Concrete to simplify Machine Learning oriented use cases.
+## Resources 
 
-## Tutorials
+### 🟨 Concrete deep dive
+- [Part I: Concrete, Zama's Fully Homomorphic Compiler](https://www.zama.ai/post/zama-concrete-fully-homomorphic-encryption-compiler)
+- [Part II: The Architecture of Concrete, Zama's Fully Homomorphic Encryption Compiler Leveraging MLIR](https://www.zama.ai/post/the-architecture-of-concrete-zama-fully-homomorphic-encryption-compiler-leveraging-mlir)
+<br></br>
 
-Various tutorials are proposed in the documentation to help you start writing homomorphic programs:
+### 🟨 Tutorials
+- [The Encrypted Game of Life in Python Using Concrete](https://www.zama.ai/post/the-encrypted-game-of-life-using-concrete-python)
+- [Encrypted Key-value Database Using Homomorphic Encryption](https://www.zama.ai/post/encrypted-key-value-database-using-homomorphic-encryption)
+- [[Video tutorial] How To Get Started With Concrete - Zama's Fully Homomorphic Encryption Compiler](https://www.zama.ai/post/how-to-started-with-concrete-zama-fully-homomorphic-encryption-compiler)
+- [[Video tutorial] Dive into Concrete - Zama's Fully Homomorphic Encryption Compiler](https://www.zama.ai/post/video-tutorial-dive-into-concrete-zamas-fully-homomorphic-encryption-compiler)
```

Comment:
- start by the Ayoub's video, then the Umut's video, then the rest, in my opinion
- also, for videos, do we want to link to blog (having videos inside), or directly the video? you do the first, I would say it's one click which is not necessary, and take the second